.github/workflows/build.yaml

```diff
@@ -318,19 +318,19 @@ jobs:
     steps:
       - name: checkout
         uses: actions/checkout@v4
-        if: ${{ !(matrix.os == 'centos' && (matrix.version != '8') || matrix.version != '11') }}
+        if: ${{ !(matrix.os == 'centos' && (matrix.version != 8 && matrix.version != 11)) }}
       - name: Set up QEMU
-        if: ${{ !(matrix.os == 'centos' && (matrix.version != '8') || matrix.version != '11') }}
+        if: ${{ !(matrix.os == 'centos' && (matrix.version != 8 && matrix.version != 11)) }}
```

Comment:
do not do this, the condition becomes tooo complicated ...

why not just start a new yaml?